### PR TITLE
#1 Simple Dockerfile and requirements.txt

### PR DESCRIPTION
**Pull request by ghost**
Here is my Dockerfile. Check it out!

### CODE REVIEW

File: oidc_example/op2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM fedora:28
```
Comment (mblomdahl) [inline oidc_example/op2/Dockerfile]: Please try using the minimal Python 3.6 base image instead.

Comment (mblomdahl) [reply oidc_example/op2/Dockerfile]: Then you won't need to install pip and stuff. :)

File: oidc_example/op2/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+future==0.17.1
+idna==2.8
+ipaddress==1.0.22
+isodate==0.6.0
+jaraco.functools==2.0
+lxml==4.3.0
+Mako==1.0.7
+MarkupSafe==1.1.0
+more-itertools==5.0.0
+oic==0.15.0
```
Comment (mblomdahl) [inline oidc_example/op2/requirements.txt]: If you merge latest changes from `fix-op2` branch into yours, you should be able to trim this requirements list down to say... `CherryPy==8.9.1` and `oic`. Try it out; 

```bash
python3 -m venv venv
. venv/bin/activate
pip install oic CherryPy==8.9.1
pip freeze > requirements.txt
```

Comment (mblomdahl) [reply oidc_example/op2/requirements.txt]: (Also add first 3 steps as a nice getting started section such as [this one](https://github.com/mkdevops-se/hq.mkdevops.se#getting-started).)

**Review by mblomdahl — CHANGES_REQUESTED** [review]

File: oidc_example/op2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.6
+COPY requirements.txt /op2/
+WORKDIR /op2/
+EXPOSE 8040
+RUN pip install -r requirements.txt
+COPY . /op2/
+ENTRYPOINT ./server.py -p 8040 -t -d config_simple.py
```
Comment (mblomdahl) [inline oidc_example/op2/Dockerfile]: @vtepliuk Do you see the annotation at the end of this line? Not good! :)

(Only visible in "Files Changed" view.)

File: oidc_example/op2/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    git clone https://github.com/OpenIDC/pyoidc.git && cd pyoidc/oidc_example/op2/  
+    python3 -m venv venv && . venv/bin/activate  
+    pip install -r requirements.txt       
+    ./start.sh 
+### Setup
+#### Client Management
+
+To be able to start up the project you have to create a new client to the client_db.
+This is done by the CLI applilcation client_manageent.py
+The allowed redirect_uris must be given and the client_id, and client_secret are generated.
```
Comment (mblomdahl) [inline oidc_example/op2/README.md]: How on earth to do you get a diff like this, when these are the only changes you made?

screenshot 2019-02-26 at 14 48 34

**Review by mblomdahl — CHANGES_REQUESTED** [review]
Look at the remarks below. Also note that the PR is conflicted.

Maybe you should create a clean new branch `docker2` from fix-op2, re-apply your changes there and open a new PR? ;)

File: oidc_example/op2/Dockerfile (hunk 3 above)
Comment (mblomdahl) [inline oidc_example/op2/Dockerfile]: This Dockerfile, seems to still not-be-working. Can you please remove it from the PR so we can merge the stuff that's accurate? Open new PR with accompanying README update when you got it fixed! :)

**Review by mblomdahl — CHANGES_REQUESTED** [review]
Remove what doesn't work, merge the rest, and come up with new proposal for Dockerfile!

File: oidc_example/op2/Dockerfile (hunk 3 above)
Comment (ghost) [reply oidc_example/op2/Dockerfile]: Fixed!

**Review by mblomdahl — APPROVED** [review]

**Pull request merged**